src/common/snippets/include/snippets/lowered/loop_info.hpp

```diff
@@ -295,6 +296,27 @@ class UnifiedLoopInfo : public LoopInfo {
      */
     void replace_with_new_ports(const ExpressionPort& actual_port, const std::vector<ExpressionPort>& target_ports) override;
 
+    /**
+     * @brief Remove the current LoopPort that contains ExpressionPort.
+     *        Note: If there is no LoopPort with ExpressionPort `ports`, does nothing.
+     *        This function remove directly without respect ports order, caller is responsible for the order by sort.
+     * @param ports need to be removed
+     */
+    void update_loop_ports(const std::vector<ExpressionPort>& actual_ports, const std::vector<ExpressionPort>& target_ports, bool is_input);
+    /**
+     * @brief Remove the current LoopPort that contains ExpressionPort.
+     *        Note: If there is no LoopPort with ExpressionPort `ports`, does nothing.
+     *        This function remove directly without respect ports order, caller is responsible for the order by sort.
+     * @param ports need to be removed
+     */
+    void remove_loop_ports(const std::vector<ExpressionPort>& ports, bool is_input);
+    /**
+     * @brief Add ports to the current LoopPort.
+     *        This function add in back of current LoopPort without respect ports order, caller is responsible for the order by sort.
+     * @param ports need to be added
+     */
+    void add_loop_ports(const std::vector<ExpressionPort>& ports, bool is_input);
```

Comment:
Let's make them private methods for now to make interface clearer

Reply:
done in [c575707](https://github.com/openvinotoolkit/openvino/pull/24044/commits/c5757078d0c1dbf4e73f824cb21c35a10b6d7cfd).Thanks Alexandra.